src/main/java/killrvideo/service/RatingsService.java

```diff
@@ -41,7 +41,6 @@
 import killrvideo.validation.KillrVideoInputValidator;
 
 @Service
-//public class RatingsService extends AbstractRatingsService {
```

Comment:
Does this have to do with a GRPC update? Wondering how it got mixed into this PR.

Reply:
It is related to GRPC update from 1.0 to 1.10, not sure why it is here in comments

Reply:
Neither was I, but since it was not being used I went an removed it.

Reply:
Makes senses to remove all that, now it's working properly with Grpc 1.10